internal/naming/names.go

```diff
@@ -478,3 +479,11 @@ func MovePGBackRestRepoDirJob(cluster *v1beta1.PostgresCluster) metav1.ObjectMet
 		Name:      cluster.Name + "-move-pgbackrest-repo-dir",
 	}
 }
+
+// UpgradeCheckConfigMap returns the ObjectMeta for the PGO ConfigMap
+func UpgradeCheckConfigMap() metav1.ObjectMeta {
+	return metav1.ObjectMeta{
+		Namespace: config.PGONamespace(),
+		Name:      "pgo-upgrade-check-configmap",
```

Comment:
:wrench: Can we drop `configmap` from the name? (feels like a bit of stutter otherwise).

:thinking: Just wondering aloud if we should name more generically - e.g. just `pgo` - as the "PGO ConfigMap" (which maybe could eventually be used for other things - though nothing else comes to mind at the moment (which is why I def don't have a problem sticking with the current naming).